snare.py

```diff
@@ -429,29 +442,53 @@ def parse_timeout(timeout):
+        "--index-page", help="file name of the index page", default='index.html')
     parser.add_argument("--port", help="port to listen on", default='8080')
     parser.add_argument("--interface", help="interface to bind to")
-    parser.add_argument("--host-ip", help="host ip to bind to", default='localhost')
-    parser.add_argument("--debug", help="run web server in debug mode", default=False)
-    parser.add_argument("--tanner", help="ip of the tanner service", default='tanner.mushmush.org')
-    parser.add_argument("--skip-check-version", help="skip check for update", action='store_true')
-    parser.add_argument("--slurp-enabled", help="enable nsq logging", action='store_true')
-    parser.add_argument("--slurp-host", help="nsq logging host", default='slurp.mushmush.org')
-    parser.add_argument("--slurp-auth", help="nsq logging auth", default='slurp')
-    parser.add_argument("--config", help="snare config file", default='snare.cfg')
-    parser.add_argument("--auto-update", help="auto update SNARE if new version available ", default=True)
-    parser.add_argument("--update-timeout", help="update snare every timeout ", default='24H')
-    parser.add_argument("--server-header", help="set server-header", default='nginx')
+    parser.add_argument(
+        "--host-ip", help="host ip to bind to", default='localhost')
+    parser.add_argument(
+        "--debug", help="run web server in debug mode", default=False)
+    parser.add_argument(
+        "--tanner", help="ip of the tanner service", default='tanner.mushmush.org')
+    parser.add_argument("--skip-check-version",
+                        help="skip check for update", action='store_true')
+    parser.add_argument("--slurp-enabled",
+                        help="enable nsq logging", action='store_true')
+    parser.add_argument("--slurp-host", help="nsq logging host",
+                        default='slurp.mushmush.org')
+    parser.add_argument(
+        "--slurp-auth", help="nsq logging auth", default='slurp')
+    parser.add_argument(
+        "--config", help="snare config file", default='snare.cfg')
+    parser.add_argument(
+        "--auto-update", help="auto update SNARE if new version available ", default=True)
+    parser.add_argument("--update-timeout",
+                        help="update snare every timeout ", default='24H')
+    parser.add_argument("--server-header",
+                        help="set server-header", default='nginx')
+    parser.add_argument(
+        "--logger_level", help="log of time-stamp, URL and source IP")
+    parser.add_argument(
+        "--log_file", help="path of the log file", default="/opt/snare/snare.log")
     args = parser.parse_args()
     base_path = '/opt/snare/'
     base_page_path = '/opt/snare/pages/'
     config = configparser.ConfigParser()
-    config.read(os.path.join(base_path,args.config))
-
+    config.read(os.path.join(base_path, args.config))
 
+    if args.logger_level:
+        if args.log_file:
```

Comment:
this is always true with the default set

Reply:
This was fixed